superset/assets/src/visualizations/FilterBox/FilterBox.css

```diff
@@ -17,55 +17,63 @@
  * under the License.
  */
 .select2-highlighted > .filter_box {
-    background-color: transparent;
-    border: 1px superset black;
+  background-color: transparent;
+  border: 1px superset black;
 }
 
 .dashboard .filter_box .slice_container > div:not(.alert) {
-    padding-top: 0;
+  padding-top: 0;
 }
 
 .input-inline {
-    float: left;
-    display: inline-block;
-    padding-right: 3px;
+  float: left;
+  display: inline-block;
+  padding-right: 3px;
 }
 
-ul.select2-results li.select2-highlighted div.filter_box{
-    color: black;
-    border-width: 1px;
-    border-style: solid;
-    border-color: #666;
+ul.select2-results li.select2-highlighted div.filter_box {
+  color: black;
+  border-width: 1px;
+  border-style: solid;
+  border-color: #666;
 }
 
-ul.select2-results div.filter_box{
-    color: black;
-    border-style: solid;
-    border-width: 1px;
-    border-color: transparent;
+ul.select2-results div.filter_box {
+  color: black;
+  border-style: solid;
+  border-width: 1px;
+  border-color: transparent;
 }
+
 .filter_box {
-    padding: 10px 0;
-    overflow: visible !important;
+  padding: 10px 0;
+  overflow: visible !important;
 }
+
 .filter_box:hover {
-    z-index: 1000;
+  z-index: 1000;
 }
+
 .m-b-5 {
-    margin-bottom: 5px;
+  margin-bottom: 5px;
 }
+
 .filter-container {
-    display: flex;
+  display: flex;
 }
+
 .filter-container label {
-    display: flex;
-    font-weight: bold;
-    margin-bottom: 8px;
+  display: flex;
+  font-weight: bold;
+  margin-bottom: 8px;
 }
+
 .filter-container .filter-badge-container {
-    width: 30px;
-    padding-right: 10px;
+  width: 30px;
+  padding-right: 10px;
 }
+
 .filter-container .filter-badge-container + div {
-    width: 100%;
+  width: 100%;
+  z-index: 11;
```

Comment:
looks like vscode applied some autoformatting. This is the only actual change.

Reply:
I noticed your colleague was adding some .less files to standardize some variables. i've found it quite useful to keep all z indexes used in an app in a single constant file that we can then import wherever it's needed (naming this one `filter-box-dropdown-layer` or something like that)

If you think that's out of scope for this PR, then maybe at least add a comment saying that we should clean that up in the future?

Reply:
@etr2460 Not out of scope, definitely happy to do it in this PR. Though it probably involved converting this file to a `.less` file. Is there some convention around using/preferring css/less/sass(?). 

`z-index`s can get pretty messy pretty quickly. At some point it may be fruitful to define hierarchies throughout the app. eg, modal > dropdowns > tooltips > regular divs. So we can use variables like `@z-index-dropdown` to abstract away the actual values.

Reply:
Yeah, defining hierarchies is something i'm definitely behind. I don't know of any preference between css/less/sass (i actually don't know why we're using all three) so i'd say you should do what you think is best here

Reply:
@etr2460 I was just going to jump into this thread, to say that exactly what you're proposing is part of my LESS refactor plan. A registry of z-index "layers" and what they're used for will be super handy to grow up with.